gatsby-browser.js

```diff
@@ -4,4 +4,4 @@
  * See: https://www.gatsbyjs.org/docs/browser-apis/
  */
 
-// You can delete this file if you're not using it
+require('./prism-theme.css')
```

Comment:
Rather can downloading and committing this file can we use a module like [`prism-themes`](https://www.npmjs.com/package/prism-themes) and instead `require('prism-themes/themes/prism-cb.css')`

Reply:
this is so we can add our own custom styles to the code blocks :+1: